src/reducers/versions.spec.tsx

```diff
@@ -1521,7 +1521,11 @@ describe(__filename, () => {
         }),
       );
       expect(dispatch).toHaveBeenCalledWith(
-        push(`${pathname}?path=${selectedPath}`),
+        push({
+          ...location,
+          search: `?path=${selectedPath}`,
+          hash: undefined,
```

Comment:
Do we need `hash: undefined` here, or are you just doing it to make the test more explicit? The code does a `delete newLocation.hash` so I wouldn't expect that our assert would need to include `hash: undefined`.

Reply:
I did that because I spread `location` above.